bin/resalloc-maint

```diff
@@ -45,6 +45,18 @@ p_t_list = subparsers.add_parser(
         "ticket-list",
         help="List not-yet-closed tickets")
 
+foreach = subparsers.add_parser(
+    "foreach-resource",
+    help="Execute command for each resource",
+)
+foreach.add_argument(
+    "--command",
+    default="echo {name}",
+    help=(
+        "Shell command to execute.  This accepts variables like "
+        "{state}, {data_utf8}, {name}, etc."),
```

Comment:
Where do I find what "etc" means? :-)

Reply:
Yeah, nowhere :-(

This functionality needs more care, and I didn't the needed time.  Better docs, more options.  Give me some time till I get back to this... (or feel free to add follow-up patches).